src/drawers/errorBarDrawer.ts

```diff
@@ -0,0 +1,16 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Drawers {
+  export class ErrorBar extends Drawer {
```

Comment:
What differentiates a line `path` and a `line` svg element...?  Not sure if `Primitive` is the right term?